client-app/app-runner.ts

```diff
@@ -0,0 +1,60 @@
+import { createApp } from "vue";
+import { createHead } from "@vueuse/head";
+import { initCfg, initContext, initMenu } from "@core/utilities";
+import { config, context, menu } from "@core/plugins";
+import App from "./App.vue";
+import blocks from "./builder-preview/pages/blocks";
+import * as yup from "yup";
+
+/**
+ * Global Styles
+ */
+import "@fortawesome/fontawesome-free/css/all.css";
+import "@/assets/styles/main.scss";
+import createI18nRouter, { currentCultureName, createI18nWithCurrentLocale } from "./i18n";
+import routes from "./router";
+
+/**
+ * Async application init
+ */
+export default async (plugins: any[] = []) => {
```

Comment:
It's better to use `Plugin` instead of `any`.

Reply:
done